dev/src/main/scala/com/gu/anghammarad/ArgParser.scala

```diff
@@ -12,44 +12,44 @@ object ArgParser {
   val argParser: OptionParser[Arguments] = new OptionParser[Arguments]("anghammarad") {
     cmd("json")
       .action { (_, _) =>
-        Json("", "", Arguments.defaultStage)
+        JsonArgs("", "", Arguments.defaultStage)
```

Comment:
As with the `Angahammarad` object, it was necessary to rename this to prevent a naming collision.